docs/design/autoware-interfaces/ad-api/list/api/planning/state.md

```diff
@@ -0,0 +1,14 @@
+# /api/planning/state
```

Comment:
I think it's better to make it more specific.

```suggestion
# /api/planning/motion_state
```

Reply:
Renamed to `/api/motion/state` because it is based on current vehicle behavior rather than planning.